Assets/FallingPlatform.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FallingPlatform : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+    public GameObject spriteObject;
+    public float elapsed = 0.0f;
+    public float duration = 1.5f;
+    public float magnitudeX = 0.3f;
```

Comment:
Out of curiosity, why would we want the platform to move sideways?

Reply:
My original idea was to allow the platforms to shake a bit sideways, but I'll set the default to 0, which gives us the option to turn it off or on .